module_utils/network_lsr/argument_validator.py

```diff
@@ -1064,6 +1064,14 @@ def _validate_post_state(self, value, name, result):
         """
         actions = []
         state = result.get("state")
+        persistent_state = result.get("persistent_state")
+        # if persistent state is present state cannot be set to present or absent
+        if persistent_state and state in ("absent", "present"):
```

Comment:
```suggestion
        if persistent_state and state in self.VALID_PERSISTENT_STATES:
```
Seems like there is a constant that you can use, just saw it after the diff.